### DISCUSSION TITLE
postal_worker_1     /docker-entrypoint.sh post ...   Exit 1 Error

### DISCUSSION THREAD

**Discussion by mail2sathishkumar008** [postalserver/postal#2211] [Help with using Postal]
Hi,

Please help me to resolve the issue.

postal_cron_1       /docker-entrypoint.sh post ...   Up
postal_requeuer_1   /docker-entrypoint.sh post ...   Exit 1
postal_smtp_1       /docker-entrypoint.sh post ...   Up
postal_web_1        /docker-entrypoint.sh post ...   Up
postal_worker_1     /docker-entrypoint.sh post ...   Exit 1

**Comment by willpower232** [comment]
You will need to look at the logs to see what the problem is.

At a guess, it is likely that your RabbitMQ was not started at the time you tried to start Postal.

**Reply by mail2sathishkumar008** [reply]
@willpower232 Appreciate your response. Please let me know how to check logs. Sorry I'm not a techie just doing all these works with the help of YouTube.

I'll share here what log is showing. So you may help me to rectify the issue.

Also I'm using Ubuntu not centos

**Reply by willpower232** [reply]
`postal logs worker` and `postal logs requeuer` should be sufficient

**Comment by lTaySoNl** [comment]
I keep getting this problem! How to solve it?
postal_cron_1 /docker-entrypoint.sh post ... Up
postal_requeuer_1 /docker-entrypoint.sh post ... Exit 1
postal_smtp_1 /docker-entrypoint.sh post ... Up
postal_web_1 /docker-entrypoint.sh post ... Up
postal_worker_1 /docker-entrypoint.sh post ... Exit 1
[log postal eror.txt](https://github.com/postalserver/postal/files/11502479/log.postal.eror.txt)

**Reply by willpower232** [reply]
the worker and requeuer require constant connection to MySQL and RabbitMQ and it seems neither of them are running for you as per the error text. Alternatively you haven't updated the config file with their details.
